pandamarket.html

```diff
@@ -0,0 +1,113 @@
+    <head>
+        <meta charset="utf-8">
+        <title>판다마켓</title>
+        <link rel="stylesheet" href="pandamarket.css">
+    </head>
+    <body>
+       <header>
+        <div class='navbar-container'>
+          <a href="pandamarket.html">
+           <img class='logo' src='./img/logo.png' alt='판다마켓로고'>
+          </a>
+          <a href="login.html">
+           <button class='login'>로그인</button>
+          </a>
+        </div>  
+       </header>
+
+
+       <main>
+        <div class="main-container">
+            <div class='main-text-btn'>
+                <h1 class='main-text'>
+                일상의 모든 물건을<br>거래해보세요
+                </h1>
+                <a href="items.html">
+                  <button class='visit-btn'>구경하러 가기</button>
+                </a>
+            </div>
+            <img src='./img/header.png' alt='logo'>
+        </div>
+       </main>
+
+
+       <div class="part1">
+        <section class='section1'>
+            <img class='left-content' src='./img/section1.png' alt='인기상품일러스트'>
+            <div class='right-content'> 
+                <p class='point'>Hot item</p>
+                <h2 class="text">인기 상품을<br>확인해보세요</h2>
+                <h2 class="desc">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</h2>
+            </div> 
+        </section>
+       </div>
+
+
+       <div class="part2">
+        <section class='section2'>
+            <div class='left-content2'> 
+                <p class='point'>Search</p>
+                <h2 class="text">구매를 원하는<br>상품을 검색하세요</h2>
+                <h2 class="desc">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</h2>
+            </div>
+            <img class='right-content2' src='./img/section2.png' alt='구매일러스트'>
+        </section>
+        </div>
+
+
+        <div class="part3">
+          <section class='section3'>
+            <img class='left-content' src='./img/section3.png' alt='판매일러스트'>
+            <div class='right-content'> 
+                <p class='point'>Register</p>
+                <h2 class="text">판매를 원하는<br>상품을 등록하세요</h2>
+                <h2 class="desc">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</h2>
+            </div>
+          </section>
+        </div>
+
+
+        <div class='empty'></div>
+
+
+       <footer>
+           <div class="foot">
+                <h2 class='text3'>믿을 수 있는<br>판다마켓 중고 거래</h2>
+                <img class='right-content3' src='./img/footer.png' alt='logo'>
+           </div>
+       </footer>
+
+       
+       <section>
+        <div class="part4">
+         <div class="underbar">
+          <p class='codeit'>@codeit - 2024</p>
+          <h3 class="ppf">
+            <a href="privacy.html">
+                <a class="privacy">Privacy Policy</a>
+            </a>
+            <a href="faq.html">
+                <a class='faq'>FAQ</a>
+            </a>
+          </h3>
+          <h3 class="logos">
+            <a href="https://www.facebook.com/?locale=ko_KR"> 
+              <img src='./img/facebook.png' alt='페이스북로고'>
+            </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
          <ul class="logos">
            <li>
              <a href="https://www.facebook.com/?locale=ko_KR"> 
                <img src='./img/facebook.png' alt='페이스북로고'>
              </a>
            </li>
```